### PR TITLE
Use [all] instead of [default] in all coafiles

### PR DESCRIPTION
**Pull request by anubhabsen**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [*] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [*] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by anubhabsen** [comment]
issue #4127 was to replace `[default]` with `[all]` in all the coafiles. But this results in some issues about the default in other file.

**Comment by thisch** [comment]
Grep for 'Default' in the source code and replace it there as well where approriate.

File: .coafile [hunk 1]
```diff
@@ -19,7 +19,7 @@ preferred_quotation = '
 default_actions = *: ApplyPatchAction
 
 [flakes]
-# Do not set default_action to ApplyPatchAction as it may lead to some
```
Comment (meetmangukiya) [inline .coafile]: unrelated change

File: .coafile [hunk 2]
```diff
@@ -44,7 +44,7 @@ files = docs/**/*.rst, CONTRIBUTING.rst, README.rst
 ignore = docs/API/**
 ignore_length_regex = ^.*https?://
 
-default_actions = SpaceConsistencyBear: ApplyPatchAction
```
Comment (meetmangukiya) [inline .coafile]: unrelated change

**Comment by meetmangukiya** [comment]
fix the tests

**Comment by Udayan12167** [comment]
There are some deprecation messages related to the `Default` section that  are checked in tests please change that. Also in some tests the sections are checked for the presence of the `default` section. It now needs to check for the `all` section.

**Comment by Udayan12167** [comment]
Please run tests before pushing and opening a PR.

**Comment by nityeshaga** [comment]
@anubhabsen You still working on this PR?

**Comment by acashmkj** [comment]
@anubhabsen  if you are still working on this issue then please write a descriptive commit message and and use Fixes(issue number) or Closes (issue number)

**Comment by jayvdb** [comment]
When you change it to `all`, the other sections also need to change.  See the configuration documentation.

**Comment by jayvdb** [comment]
Closing as abandoned.
